packages/extension-ui/src/Popup/ImportSeed/SeedAndPath.tsx

```diff
@@ -177,10 +183,30 @@ function SeedAndPath({ className, genesis, onAccountChange, onNextStep, type }:
     </CustomFooter>
   );
 
+  const onSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (isFormValid) {
+      onNextStep();
+
+      return;
+    }
+
+    if (error) {
+      return;
+    }
+
+    setError(t('This field is required'));
```

Comment:
Should an error really be set in here, after `if (error) return`? Not saying it's wrong, I'm just suspicious :P

Reply:
yes, this is correct, but I'll change the structure here :sweat_smile: 
Since isFormValid = address && !error
and isFormValid = false
and error === false
it means address is falsy, but that's unnecessarily complicated

Reply:
Yeah, I think more explicitness in here would be useful. Not only for readability, but it'll also guard ass from refactor errors when someone adds another compound to the `isFormValid` logical expression, making the current assumption incorrect.